### PR TITLE
Add nic and bluetooth device types

### PR DESCRIPTION
**Pull request by rneilturner**
Add support to vusb for filtering on both bluetooth and nic devices

### CODE REVIEW

**Comment by rneilturner** [comment]
I would like to revisit the changes proposed by Troy four years ago. I will be taking it from here in regards to any recommended changes so I decided to create a new PR. If the community would prefer me taking over the PR Troy did let me know I will cancel this PR.

File: src/db.c [hunk 1]
```diff
@@ -383,6 +401,12 @@ db_write_policy(rule_t *rules)
         db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_MASS_STORAGE, "1");
       if (rule->dev_type & OPTICAL)
         db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_OPTICAL, "1");
+      if (rule->dev_type & NIC)
+        db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_NIC, "1");
+      if (rule->dev_type & BLUETOOTH)
+        db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_BLUETOOTH, "1");
+
+
```
Comment (jandryuk) [inline src/db.c:409]: stray newlines.

File: src/db.c [hunk 2]
```diff
@@ -393,6 +417,11 @@ db_write_policy(rule_t *rules)
         db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_GAME_CONTROLLER, "0");
       if (rule->dev_not_type & MASS_STORAGE)
         db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_MASS_STORAGE, "0");
+      if (rule->dev_not_type & NIC)
+        db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_NIC, "0");
+      if (rule->dev_not_type & BLUETOOTH)
+        db_write_rule_key(rule->pos, NODE_DEVICE "/" NODE_BLUETOOTH, "0");
+
```
Comment (jandryuk) [inline src/db.c:424]: stray newline.

**Review by jandryuk — COMMENTED** [review]
Using this PR is fine, but it looks like the previous PR feedback wasn't fully addressed.  This one is simple:
https://github.com/OpenXT/vusb-daemon/pull/14#discussion_r110377798

The larger feedback concerned the heuristics in this patch and false positives and negatives.  Do you have any comment on that?

**Comment by tsirakisn** [comment]
@jandryuk this was replaced by https://github.com/OpenXT/vusb-daemon/pull/33 and can be closed

**Pull request closed without merging**